moabb/datasets/braininvaders.py

```diff
@@ -704,3 +770,88 @@ def data_path(
         self, subject, path=None, force_update=False, update_path=None, verbose=None
     ):
         return _bi_data_path(self, subject, path, force_update, update_path, verbose)
+
+
+class VirtualReality(BaseDataset):
+    """
+    We describe the experimental procedures for a dataset that we have made publicly
```

Comment:
Can you add the data summary to the doc, please? 

Example from BNCI2014008.
    .. admonition:: Dataset summary


        ===========  =======  =======  =================  ===============  ===============  ===========
        Name           #Subj    #Chan  #Trials / class    Trials length    Sampling rate      #Sessions
        ===========  =======  =======  =================  ===============  ===============  ===========
        BNCI2014008        8        8  3500 NT / 700 T    1s               256Hz                      1
        ===========  =======  =======  =================  ===============  ===============  ===========